### DISCUSSION TITLE
sync failure

### DISCUSSION THREAD

**Discussion by Alone0316** [ROM-builders/temporary#12937] [General]
https://cirrus-ci.com/build/5984331570085888 this guy bdjmfoe is making riceDroid and Lineage and tracking that clang version 416xx leading to failure of all other builds which are being triggered after him

**Comment by Apon77** [comment]
What can I do?

**Reply by Alone0316** [reply]
You can ask him to not clone that clang and use the default one as because of him cloning that clang his build is succeeding but all others are failing which are being triggered after him or either something can be done from local manifest or build_rom.sh like removing that clang and syncing the default one which comes with the source itself?
